src/main/java/com/deque/axe/android/AxeView.java

```diff
@@ -93,11 +94,26 @@
    */
   public final String viewIdResourceName;
 
+  /**
+   * Is view an instance of WebView.
+   */
+  private final boolean isWebView;
+
+  /**
+   * Is view a child of a WebView.
+   */
+  private final boolean isWebViewChild;
```

Comment:
I feel like this should be renamed isInWebView. Also, I think we should just make it a function, I don't think the performance benefit of avoiding a  O(N) operation is worth serializing this.

We should at least make it transient.

Reply:
Or we can just remove this altogether and assume anything that is a child of a webview is in the webview.